src/win/winapi.c

```diff
@@ -166,4 +170,11 @@ void uv_winapi_init(void) {
       GetProcAddress(user32_module, "SetWinEventHook");
   }
 
+  iphlpapi_module = LoadLibraryA("iphlpapi.dll");
```

Comment:
Does that mean that `-liphlpapi` can be dropped from configure.ac and uv.gyp?  Or is there more code using symbols from that dll directly?

Reply:
No, we can not remove it. because we have used `GetAdaptersAddresses` coming from `iphlpapi`.